includes/Admin/Settings.php

```diff
@@ -312,4 +340,70 @@ public function get_tabs() {
 		 */
 		return (array) apply_filters( 'wc_facebook_admin_settings_tabs', $tabs, $this );
 	}
+
+	/**
+	 * Add the Facebook for WooCommerce tabs to the Facebook Product Set taxonomy page.
+	 * Renders the tabs (hidden by default) at the stop of the page,
+	 * then moves them to the correct DOM location with JavaScript and displays them.
+	 *
+	 * @since 3.3.0
+	 */
+	public function add_tabs_to_product_sets_taxonomy() {
+		// Only load this on the edit-tags.php page
+		$screen                  = get_current_screen();
+		$is_taxonomy_list_page   = 'edit-tags' === $screen->base;
+		$is_taxonomy_term_page   = 'term' === $screen->base;
+		$is_taxonomy_page        = $is_taxonomy_list_page || $is_taxonomy_term_page;
+		$is_product_set_taxonomy = 'fb_product_set' === $screen->taxonomy && $is_taxonomy_page;
+
+		if ( $is_product_set_taxonomy ) {
+			$this->render_tabs( Settings_Screens\Product_Sets::ID );
+			?>
+				<style>
+					.facebook-for-woocommerce-tabs {
+						margin-top: 20px;
+						margin-bottom: 10px;
+						display: none;
+					}
+					#wpbody-content > .wrap > h1 {
+						margin-top: 65px;
+						font-size: 1.3em;
+						font-weight: 600;
+					}
+					<?php
+					// On the taxonomy list page in mobile view, account for the Screen Options tab.
+					if ( $is_taxonomy_list_page ) :
+						?>
+					@media (max-width: 782px) {
+						.facebook-for-woocommerce-tabs {
+								padding-top: 0;
+								position: relative;
+								top: -10px;
+								margin-bottom: -10px;
+							}
+						}
+					<?php endif; ?>
+				</style>
+				<script>
+						document.addEventListener('DOMContentLoaded', function() {
```

Comment:
@layoutd 
This approach works, but out of curiosity, have you considered inserting the tabs using the `all_admin_notices` action? It might be a cleaner solution, as it ensures the tabs are added after the screen options and notices, avoiding the need for addEventListener for DOM manipulation.

Reply:
ZOMG thank you! I tried a few times to remember the exact hook to use, but even looking at [Actions Run During an Admin Page Request](https://developer.wordpress.org/apis/hooks/action-reference/#actions-run-during-an-admin-page-request) all my brain could provide was `admin_notices`, which didn't work.
I've made the change in ea089fa7f9cd0f4a18fd715099ee0795ddb69bd3 and cleaned up the JS (and CSS a bit, too)